taccsite_cms/static/site_cms/css/src/_imports/trumps/s-drop-cap.css

```diff
@@ -1,5 +1,15 @@
-@import url('../tools/x-drop-cap.css');
+@supports (initial-letter: 3){
+  .s-drop-cap > p:first-of-type::first-letter {
+    padding-right: 1em; /* approx: `1em` of normal text */
+    initial-letter: 3;
+  }
+}
 
-.s-drop-cap > p:first-of-type::first-letter {
-  @extend .x-drop-cap;
+@supports (-webkit-initial-letter: 3) {
+  .s-drop-cap > p:first-of-type::first-letter {
+    padding-right: 1em; /* approx: `1em` of normal text */
+    margin-left: -1em; /* remove apprx. space on left side of text */
```

Comment:
Ahh gotcha. I'll remove this. I was concerned with the tiny bit of padding you see on the drop-cap there without the margin. But if we're okay with it, there's no reason to put it in.

Reply:
Sensible. I notice the tiny bit of padding. M.S. does very much like to align things…

But my theory is that the extra space is built into the letter of that font, and any such extra space may be different per letter and per font (especially given Safari's incomplete implementation), thus attempts to avoid the space will not always work. Feel free to test the theory, offer alternatives, or ask M.S. what he'd prefer.

Reply:
I wanted to make sure I wasn't making stuff up. I tested my theory on just Arial font. Different letters need different love.

| D | Z | A | P |
| - | - | - | - |
| <img width="708" alt="D" src="https://github.com/TACC/Core-CMS/assets/62723358/a0f5517e-185e-498c-8a18-686ad1a2d5fa"> | <img width="708" alt="Z" src="https://github.com/TACC/Core-CMS/assets/62723358/f956578b-a932-407e-9a2f-e3b85bc14325"> | <img width="708" alt="A" src="https://github.com/TACC/Core-CMS/assets/62723358/70c1c12c-9c0e-4490-9446-ad06a31ac9f9"> | <img width="708" alt="P" src="https://github.com/TACC/Core-CMS/assets/62723358/83d4e709-f9ca-40f1-9bd4-69f360e36915"> |